tests/internal/network/test_server.py

```diff
@@ -19,17 +21,18 @@ def setUp(self):  # noqa: N802
 
     def test_init_calls_create_server_socket(self):
         network.Server.__init__(
-            self.mock, sentinel.host, sentinel.port, sentinel.protocol)
+            self.mock, sentinel.host, 1234, sentinel.protocol)
```

Comment:
Yeah for me too. I believe it was because I added integer checking [here](https://github.com/mopidy/mopidy/pull/1629/files#diff-08f64433e79c2e70c88794e03241faceR109) as part of making the port optional but ensuring its an integer when specified

Reply:
That makes sense, thanks. I guess having validation down in this code is a bit unusual for us, I don't think we have anywhere else where the config setting's requirement is conditional on another bit of our config. For another day, I think.